specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/amlFilesystems_Archive.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "resourceGroupName": "scgroup",
+    "cacheName": "sc",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-07-23-preview",
+    "archiveInfo" : {
+      "filesystemPath" : "/mnt/fs1"
```

Comment:
This looks like a client-side path. I don't see how this makes sense to us.

Reply:
Any suggestions?
Also, any suggestions for the description in the input parameter description.

amlFilesystemArchiveInfo": {
      "type":"object",
      "description": "Information required to execute the archive operation",
      "properties": {
        "filesystemPath" : {
          "type" : "string",
          "description" : "The path to the filesystem to archive."
        }
      }
    },

Reply:
We need BrianB/Joe to weigh in here

Reply:
You can specify a directory that you want to archive, if you want to only archive a subset of the data.

Instead of filesystemPath, maybe lustrePath makes more sense since the path is relative to the root of the lustre file system (without the client mount point)?

Reply:
Path relative to the root of the Lustre filesystem makes a lot of sense to me. Need to say that explicitly in the doc string and modify the example(s) appropriately.

Reply:
Updated field name to lustrePath.
Updated description to "Path of the data to archive relative to the root of the Lustre file system.  Specify '/' to archive all modified data."
Updated the examples to "/"

Reply:
This shows up as filesystemPath for me, but that's fine - I prefer filesystemPath over lustrePath. (Lustre is properly capitalized, and going with filesystemPath avoids camelCase issues)

Reply:
Fixed the examples in this one to "/" to match the others.  Updated all to filesystemPath.
Should be in next commit.